src/main/webapp/dev/js/main/instructorFeedbacks.js

```diff
@@ -61,49 +61,19 @@ function isTimeZoneIntialized() {
 }
 
 /**
- * Format a number to be two digits
- */
-function formatDigit(num) {
-    return (num < 10 ? '0' : '') + num;
-}
-
-/**
- * Format a date object into DD/MM/YYYY format
- * @param date
- * @returns {String}
- */
-function convertDateToDDMMYYYY(date) {
-    return `${formatDigit(date.getDate())}/${formatDigit(date.getMonth() + 1)}/${date.getFullYear()}`;
-}
-
-/**
- * Format a date object into HHMM format
- * @param date
- * @returns {String}
- */
-function convertDateToHHMM(date) {
-    return formatDigit(date.getHours()) + formatDigit(date.getMinutes());
-}
-
-/**
- * To be run on page finish loading, this will select the input: start date,
- * start time, and timezone based on client's time.
+ * To be run on page finish loading, timezone based on client's time.
```

Comment:
I feel the original comment contains more details that are still applicable. Restore it?

Reply:
Yes, I forgot to revert the comment to original. Thanks for pointing out.